app/templates/gentelella/guest/event/_ticketing_box.html

```diff
@@ -270,9 +270,9 @@ <h3 class="panel-title">{{ _("Ticket Information") }}</h3>
         var discount_amount = -1;
         var promo_code_status = 0;
         var discount_code = "";
-        {% if code %}
+        {% if discount_code %}
```

Comment:
Tell me this. Where are you checking for access_code in this page? I don't see any access_code related code here. So how is access_code working. Please explain.